src/System.Private.ServiceModel/src/project.json

```diff
@@ -1,58 +1,58 @@
 {
   "dependencies": {
-    "System.Collections": "4.0.10-beta-*",
-    "System.Collections.Concurrent": "4.0.10-beta-*",
-    "System.Collections.NonGeneric": "4.0.0-beta-*",
-    "System.Collections.Specialized": "4.0.0-beta-*",
-    "System.ComponentModel": "4.0.0-beta-*",
-    "System.ComponentModel.EventBasedAsync": "4.0.10-beta-*",
-    "System.Diagnostics.Contracts": "4.0.0-beta-*",
-    "System.Diagnostics.Debug": "4.0.10-beta-*",
-    "System.Diagnostics.Tools": "4.0.0-beta-*",
-    "System.Globalization": "4.0.10-beta-*",
-    "System.IO": "4.0.10-beta-*",
-    "System.IO.Compression": "4.0.0-beta-*",
-    "System.Linq": "4.0.0-beta-*",
-    "System.Linq.Expressions": "4.0.10-beta-*",
-    "System.Linq.Queryable": "4.0.0-beta-*",
-    "System.Net.Http": "4.0.0-beta-*",
+    "System.Collections": "4.0.10",
+    "System.Collections.Concurrent": "4.0.10",
+    "System.Collections.NonGeneric": "4.0.0",
+    "System.Collections.Specialized": "4.0.0",
+    "System.ComponentModel": "4.0.0",
+    "System.ComponentModel.EventBasedAsync": "4.0.10",
+    "System.Diagnostics.Contracts": "4.0.0",
+    "System.Diagnostics.Debug": "4.0.10",
+    "System.Diagnostics.Tools": "4.0.0",
+    "System.Globalization": "4.0.10",
+    "System.IO": "4.0.10",
+    "System.IO.Compression": "4.0.0",
+    "System.Linq": "4.0.0",
+    "System.Linq.Expressions": "4.0.10",
+    "System.Linq.Queryable": "4.0.0",
+    "System.Net.Http": "4.0.0",
     "System.Net.NameResolution": "4.0.0-beta-*",
```

Comment:
It's a bit odd this one is unmodified and still reflects beta-*.  Any ideas why?